apps/conf/credentials/cosign-cred.yaml

```diff
@@ -0,0 +1,10 @@
+name: cosign-cred-test
```

Comment:
why "test" added with all names ?

it should be same as hard-coded names which removed